lib/client/fast_render.js

```diff
@@ -18,10 +18,17 @@ if(FastRender._disable) {
 //  This is the cure
 FastRender.injectDdpMessage = function(conn, message) {
   FastRender["debugger"].log('injecting ddp message:', message);
-  var originalWait = conn._waitingForQuiescence;
-  conn._waitingForQuiescence = function() {return false};
-  conn._livedata_data(message);
-  conn._waitingForQuiescence = originalWait;
+  if (conn._bufferedWrites) {
```

Comment:
I think this change should not have been included because if the connection is `_waitingForQuiescence` (for instance if a login method is run on startup), the writes get placed into a `_messagesBufferedUntilQuiescence` queue.

Those messages will not get into the `_bufferedWrites` queue until quiescence happens (the method returns from the server). So the later `_flushBufferedWrites` will not pick them up.